File: src/Main/Models/teacher.py
from django.db import models

class Teacher(models.Model):
    name = models.CharField(max_length = 128)
    cnp = models.IntegerField()
    date_of_birth = models.DateField()
    mail_address = models.CharField(max_length = 128)
    phone_number = models.CharField(max_length = 10)

    def __str__(self):
        return f"{self.cnp} - {self.name} - {self.date_of_birth} - {self.mail_address} - {self.phone_number}"

File: uni-fe/src/models/Teacher.ts
import { User } from "./User";

export interface Teacher {
    id: number,
    name: string,
    cnp: number,
    date_of_birth: string,
    phone_number: string,
    mail_address: string,
    description: string,
    courses_count: number,
    added_by: User
}